src/main.R

```diff
@@ -0,0 +1,26 @@
+library(dplyr)
+
+source(here::here("src/data_processing.R"))
+source(here::here("src/auto_scaling_algorithm.R"))
+source(here::here("src/calculate_adi.R"))
+
+util_data <- processing_data("data/util_data.csv")
+decreasing_data <- read.csv("data/decreasing_data.csv")
```

Comment:
this `decreasing_data` is meant to be the main input data, right? shouldn't we receive it via Args?

Reply:
Done. Now input data is received via args